app/components/home.jsx

```diff
@@ -43,12 +43,11 @@ export class Home extends Component {
             <div className="sub">
               <h2>A tool for tracking your Living Dex progress! We currently support:</h2>
               <ul>
+                <li><h2><Link className="link" alt="Sample Generation 8 National Living Dex" to="/u/ashketchum10/sword-national-living-dex">Generation 8 National Dex</Link> (and Generations 7 + 6)</h2></li>
```

Comment:
i decided to combine these cuz the list was getting long, thoughts?

Reply:
works for me! thoughts on making it `6 + 7` vs `7 + 6`? it reads a bit better that way